### ISSUE TITLE
add italic,bold, italicBold fonts - got error

### ISSUE THREAD

**Issue by 4t4nner** [jobsta/reportbro-lib#13]
If i try add multiple font files
```
additional_fonts = [
        dict(
            value='OpenSans', 
            filename=assetFontPath + 'OpenSans-Regular.ttf',
            bold_italic_filename=assetFontPath + 'OpenSans-BoldItalic.ttf',
            bold_filename=assetFontPath + 'OpenSans-Bold.ttf',
            italic_filename=assetFontPath + 'OpenSans-Italic.ttf'
        )
    ]
```
I got an error "undefined font opensans"
I debug into lib code and find that in [set_font](https://github.com/jobsta/reportbro-lib/blob/87e2bfc1c151d6aba700137ab75d58e9a49f1bcd/reportbro/reportbro.py#L440) added only first `opensansB` font and `font['added']` set to `True`, but `opensans` font and other not added.

Then I get [this error](https://github.com/reingart/pyfpdf/blob/cb3340806c140b6e63eb4ce9d6230be84fba456e/fpdf/fpdf.py#L628) 

if I change lib code like this:
```
                if not font['added'].get(family + '_' + style):
                    filename = font['style2filename'].get(style)
                    self.add_font(family, style=style, fname=filename, uni=font['uni'])
                    font['added'][family + '_' + style] = True
```
all works

**Comment by alhman** [comment]
This is indeed a bug, your fix is ok (assuming you changed 'added' from bool to dict). This will be fixed with the next release.

**Comment by alhman** [comment]
Fixed with 932b0be4ac1
